web_domain_field/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Web Domain Field",
+    "summary": """
+        Use computed field as domain""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://acsone.eu/",
+    "depends": ["web"],
+    "data": ["views/web_domain_field.xml"],
+    "demo": [],
```

Comment:
@Laurent-Corron If not needed then remove.

Reply:
Done!